lua/auto-save/config.lua

```diff
@@ -34,13 +34,22 @@ function Config:handle_deprecations(custom_opts)
     custom_opts["execution_message"] = nil
   end
 
+  if custom_opts["cancel_defered_save"] then
```

Comment:
`cancel_defered_save` isn't a top level setting. Should this check for `custom_opts["trigger_events"]["cancel_defered_save"]` instead?

Reply:
I just tested this. It does not react to `cancel_defered_save` inside `trigger_events` as it should but does react to a top-level `cancel_defered_save` so the table level isn't right.

Reply:
Ohh, my bad. I fixed it.